run_interesting_forecasting_api_zap_tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+environment="local"
+tags="not @smoke and not @ignore"
+if [ $# -gt 0 -a "$1" != "$environment" ];
+then
+  environment="$1"
+  tags="not @ignore"
+fi
+
+echo "*** running on $environment for tags '$tags' ***"
+
+sbt -Denvironment="$environment" -Dcucumber.options="--tags '$tags'" clean 'testOnly uk.gov.hmrc.test.cucumber.runners.InterestForecastingApiTestRunner'
```

Comment:
Does the IFS acceptance tests run as part of the Zap tests?

Reply:
zap run after  IFS acceptance tests

Reply:
zap proxy listens to IFS acceptance tests running and then it does a scan after that wards